### DISCUSSION TITLE
Adding Some Flexiglass Features

### DISCUSSION THREAD

**Discussion by Jimmbo** [rxhanson/Rectangle#988] [Ideas]
Hey, Ryan, I was following this discussion on Macupdate, and wonder whether you saw the response, and whether it's something you might be considering for Rectangle? FWIW, I'm not the person you're talking to there, just a lurker :)
<img width="625" alt="Screen Shot 2022-11-06 at 6 23 30 PM" src="https://user-images.githubusercontent.com/7252766/200201223-dbaa78d1-782f-4aba-808b-3e6bf6f185cc.png">

**Comment by rxhanson** [comment]
Thanks for the follow up. In short, these are features that are not high on my priority.

Modifier + cursor move to move/resize windows is built into [Rectangle Pro](https://rectangleapp.com/pro), and I don't have plans to add that into Rectangle. In code and in configuration, it just fits better there and Rectangle is intended to be really slim for configuration (as in, Rectangle is supposed to be an app simple enough that my mom could figure it out without too much help).

The stoplight button behavior gets requested enough that I'd be fine merging a good pull request for it, but I don't have plans to add it in myself.

**Reply by Jimmbo** [reply]
Thanks for reply. Fine by me if it's in Rectangle Pro, that's what I'd opt for anyway (I'm not yet a customer; am shopping around.....speaking of which, are you aware of any comparisons out there of SizeUp vs Rectangle/Rectangle Pro?)

**Reply by rxhanson** [reply]
I'm pretty sure SizeUp still only includes keyboard shortcuts and not any other kind of snapping behavior. SizeUp does have move across Spaces actions that are not in Rectangle/Pro because there isn't an Apple supported way to do this and I can't guarantee that Spaces actions will work properly for everyone.
